deploy/task-cad-checks.yaml

```diff
@@ -5,34 +5,34 @@ metadata:
   namespace: configuration-anomaly-detection
 spec:
   params:
-  - name: payload
-    type: string
-    description: Json string of the event data
+    - name: payload
+      type: string
+      description: Json string of the event data
   steps:
-  - name: check-infrastructure
-    # this sha is broken, should fix when we onboard
-    image: quay.io/app-sre/cadctl@sha256:22d2a957d935e883f45addd35acc87450ce71ea2f94c33f2df70ff36b015486a
-    command:
-    - "/bin/bash"
-    - "-c"
-    args:
-    - |-
-      # save the payload to a file
-      folder=$(mktemp -d)
-      file=${folder}/payload.json
-      echo '$(params.payload)' > $file
-      # run the cadctl command
-      RESULT=$(cadctl cluster-missing --payload-path $file )
-    env:
-    - name: CAD_ESCALATION_POLICY
-      value: XXXX
-    - name: CAD_SILENT_POLICY
-      value: XXXX
-    # envFrom should pull all of the secret information as envvars, so key names should be uppercase
-    envFrom:
-    - secretRef:
-        name: cad-aws-credentials
-    - secretRef:
-        name: cad-ocm-client-secret
-    - secretRef:
-        name: cad-pd-token
+    - name: check-infrastructure
+      # this sha is broken, should fix when we onboard
+      image: quay.io/app-sre/cadctl@sha256:22d2a957d935e883f45addd35acc87450ce71ea2f94c33f2df70ff36b015486a
```

Comment:
Oh one thing: Do we want to set the correct image name here? the image is located at: quay.io/app-sre/configuration-anomaly-detection

Reply:
and the tag should be used instead of the sha i guess

Reply:
It is, in the openshift template

Reply:
I think this is resolved... as long as it is in the openshift template. We should also make sure to use the openshift template for dev deployments in the future. Today, we used the deploy folder :thinking: although both should be the same, right?

Reply:
Yes